src/Collectors/CollectedData.php

```diff
@@ -8,6 +8,8 @@
 
 /**
  * @api
+ *
+ * @phpstan-type CollectorData = array<string, array<class-string<Collector<Node, mixed>>, list<mixed>>>
```

Comment:
IIRC this `=` is too much

```suggestion
 * @phpstan-type CollectorData array<string, array<class-string<Collector<Node, mixed>>, list<mixed>>>
```

Reply:
= works and importantly, it's the only syntax recognised by PhpStorm.

Reply:
I had that at first, but then aligned it with existing `@phpstan-type LinesToIgnore = array<string, array<int, non-empty-list<string>|null>>`

Reply:
@ondrejmirtes maybe we should use it in the docs then?

https://phpstan.org/writing-php-code/phpdoc-types#local-type-aliases

Reply:
I'd rather be if PhpStorm fixed that on their side.

Reply:
At least something work in PHPStorm without =:

![image](https://github.com/user-attachments/assets/1cfce0ec-b85f-4291-9035-101579338df3)